### PR TITLE
2275 - Add a database migration for the survey response

### LINKED ISSUE #2275: Create database migration
## Survey

Table `survey_responses` columns
- response_id : uuid
- submission_id: uuid
- response : JSONB[]

Example....
{
 questions: { 1: "what is ...?", 2: "why....?" }
 answers: { 1: "this" , 2: "that" }
}

### PR DESCRIPTION
**Pull request by petereast**
#### Background

What does this PR do?
Closes #2275 

#### Any relevant tickets

Please use the phrase 'Closes' if it will fully complete the ticket.

#### How has this been tested?
**Reviewers** ensure that these test requirements are also reviewed.

Please indicate the need for this change to be tested. If not please justify, if so then at what levels. 

- [ ] Unit / Integration tests
- [ ] Browser tests
- [ ] End2end tests

#### UI Changes

- [ ] Has been reviewed against Designs
- [ ] Necessary updates to styleguide

### CODE REVIEW

File: packages/component-model/migrations/1562838080-survey-response.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE survey_response (
+  response_id UUID PRIMARY KEY,
```
Comment (diversemix) [inline packages/component-model/migrations/1562838080-survey-response.sql]: we need some sort of survey id ?

Comment (petereast) [reply packages/component-model/migrations/1562838080-survey-response.sql]: yes

**Review by Cooryd — APPROVED** [review]

**Comment by petereast** [comment]
Is this all we need to do for this PR? https://github.com/elifesciences/elife-xpub/issues/2275#issuecomment-510120638

**Comment by Cooryd** [comment]
#2292 has been raised for the ORM class to be created so this PR adding the migration should be enough

**Comment by petereast** [comment]
cool :+1:

**Review by will-byrne — APPROVED** [review]

File: packages/component-model/migrations/1562838080-survey-response.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE survey_response (
+  id UUID PRIMARY KEY,
+  survey_id UUID NOT NULL,
+  submission_id UUID NOT NULL,
+  response JSONB NOT NULL,
+  created TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT current_timestamp
```
Comment (diversemix) [inline packages/component-model/migrations/1562838080-survey-response.sql:6]: If this has been added so we can use the ORM ... where is the `updated` field?

**Pull request merged**